STEERING-COMMITTEE.md

```diff
@@ -294,20 +323,69 @@ one week for comments and questions before a vote will occur.
+| <img width="30px" src="https://github.com/dewitt.png">         | Dewitt Clinton       | [@dewitt](https://github.com/dewitt)                 | Bootstrap  | 2019     |
+| <img width="30px" src="https://github.com/mchmarny.png">       | Mark Charmny         | [@mchmarny](https://github.com/mchmarny)             | Bootstrap  | 2019     |
+| <img width="30px" src="https://github.com/jdumars.png">        | Jaice Singer DuMars  | [@jdumars](https://github.com/jdumars)               | 2019       | 2020     |
+| <img width="30px" src="https://github.com/lindydonna.png">     | Donna Malayeri       | [@lindydonna](https://github.com/lindydonna)         | 2019       | 2020     |
+| <img width="30px" src="https://github.com/rgregg.png">         | Ryan Gregg           | [@rgregg](https://github.com/rgregg)                 | Bootstrap  | 2020     |
+| <img width="30px" src="https://github.com/isdal.png">          | Tomas Isdal          | [@isdal](https://github.com/isdal)                   | Bootstrap  | 2020     |
+| <img width="30px" src="https://github.com/anicolao.png">       | Alex Nicolau         | [@anicolao](https://github.com/anicolao)             | 2020       | 2020     |
+| <img width="30px" src="https://github.com/ronavn.png">         | Ron Avnur            | [@ronavn](https://github.com/ronavn)                 | 2020       | 2020     |
+| <img width="30px" src="https://github.com/bsnchan.png">        | Brenda Chan          | [@bsnchan](https://github.com/bsnchan)               | Bootstrap  | 2021     |
+| <img width="30px" src="https://github.com/mbehrendt.png">      | Michael Behrendt     | [@mbehrendt](https://github.com/mbehrendt)           | Bootstrap  | 2021     |
+| <img width="30px" src="https://github.com/vaikas.png">         | Ville Aikas          | [@vaikas](https://github.com/vaikas)                 | 2020       | 2022     |
+| <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie           | [@pmorie](https://github.com/pmorie)                 | 2020       | 2022     |
+| <img width="30px" src="https://github.com/thisisnotapril.png"> | April Kyle Nassi     | [@thisisnotapril](https://github.com/thisisnotapril) | Bootstrap  | 2022     |
+| <img width="30px" src="https://github.com/itsmurugappan.png">  | Murugappan Chetty    | [@itsmurugappan](https://github.com/itsmurugappan)   | 2022       | 2023     |
+| <img width="30px" src="https://github.com/csantanapr.png">     | Carlos Santana       | [@csantanapr](https://github.com/csantanapr)         | 2021       | 2023     |
+| <img width="30px" src="https://github.com/lance.png">          | Lance Ball           | [@lance](https://github.com/lance)                   | 2021       | 2023     |
+| <img width="30px" src="https://github.com/puerco.png">         | Adolfo García Veytia | [@puerco](https://github.com/puerco)                 | 2023       | 2024     |
+
+### Technical Oversight Committee
+
```

Comment:
We might want to add a section here about what the "TOC" actually _was_ as this is the first mention of the (obsolete) TOC.

Reply:
Added some text here: https://github.com/knative/community/pull/1587/files#r1688479540